pilot/pkg/networking/core/listener_inbound.go

```diff
@@ -324,13 +328,15 @@ func (lb *ListenerBuilder) inboundChainForOpts(cc inboundChainConfig, mtls authn
 				Filters:          lb.buildInboundNetworkFiltersForHTTP(cc),
 				TransportSocket:  buildDownstreamTLSTransportSocket(opt.ToTransportSocket(mtls)),
 				Name:             cc.Name(opt.Protocol),
+				Metadata:         util.BuildConfigInfoMetadata(cc.configMetadata),
```

Comment:
I don't see anywhere where we are skipping adding this if config.Meta is empty?

```
    metadata:
      filterMetadata:
        istio:
          config: /apis///namespaces///
```
is what I get